src/scss/components/file/mixins/hxFileControl.scss

```diff
@@ -0,0 +1,66 @@
+@import "vars";
+
+@mixin hxFileControl(
+  $pseudo-state: null,
+  $btnState: null,
```

Comment:
Please remove `$btnState` argument. This mixin is not responsible for applying those styles.

Reply:
Changes have been done, please review.